src/main/scala/mesosphere/marathon/core/event/impl/stream/HttpEventStreamServlet.scala

```diff
@@ -116,6 +116,11 @@ class HttpEventStreamServlet(
     @volatile private var handler: Option[HttpEventSSEHandle] = None
 
     override def onOpen(emitter: Emitter): Unit = {
+      // We don't want to count this response towards the http metrics as it could be quite large by the time it closes.
+      // Also, the serialization for events is done once for all consumers; we should track the data with a separate
+      // metric.
+      HttpTransferMetricsHandler.exclude(request)
```

Comment:
Good point! Created [MARATHON-8264](https://jira.mesosphere.com/browse/MARATHON-8264) to keep track of this.